content/extra.md

```diff
@@ -8,18 +8,18 @@ include = "extra"
 
 runC does not create a daemon, so it integrates well with systemd.
 
-```
+```systemd
 [Unit]
 Description=Minecraft Build Server
 Documentation=http://minecraft.net
 After=network.target
 
 [Service]
-CPUQuota=200%
-MemoryLimit=1536M
-ExecStart=/usr/local/bin/runc
-Restart=on-failure
-WorkingDirectory=/containers/minecraftbuild
+Type=forking
+ExecStart=/usr/sbin/runc run -d --pid-file /run/minecraft.pid minecraft-build-container
+ExecStopPost=/usr/sbin/runc delete minecraft-build-container
```

Comment:
Probably. Though I also should add a note about installing `runc` from distributions, since in distributions it's installed at `/usr/sbin/runc`.